### ISSUE TITLE
Informative note for Unicode in EPUB accessibility

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/epub-specs#1790]
Using Unicode text is many times considered as implicit requirement. Furthermore it does not effect English speaking countries.
But the fact is that you can find non-Unicode web content in non-English speaking parts of the world. The problem is even more in E-Publications because you can embed fonts in the publications.
So, we should at least have an informative text in EPUB Accessibility specifications which remind the publishers to use Unicode text.

**Comment by mattgarrish** [comment]
@avneeshsingh It's a requirement of the EPUB specification to use UTF-8, and we have a technique already about not using [images of text](https://w3c.github.io/epub-specs/epub33/a11y-tech/#text-001).

Does this cover your concerns?

**Comment by avneeshsingh** [comment]
@mattgarrish it is great that it is among basic requirements of EPUB. I get this question from people who read EPUB accessibility but do not go through complete EPUB 3.x specs.
There are places where mainly EPUB accessibility would be visible, for example we expect EPUB Accessibility to become a technical specification under EU accessibility directive, similarly I had proposed EPUB accessibility to BIS India.

So, it would be good to have a line about this in EPUB Accessibility which refers to the original text of Unicode requirement in EPUB 3.3 main specs.

**Comment by mattgarrish** [comment]
> So, it would be good to have a line about this in EPUB Accessibility which refers to the original text of Unicode requirement in EPUB 3.3 main specs.

The problem is I'm not sure where we add something like this. The only place where we talk about epub is in the epub requirements section, but there you would have to make this a new requirement of the specification, which would in turn be redundant since you can't create EPUBs any other way..

We'd seem to need a new section or appendix, but I'm not sure exactly what that section or appendix would be covering that would allow this to be a part of.

**Comment by murata2makoto** [comment]
I might sidetrack, but I would like the A11Y spec to explicitly state that different languages and cultures require different accessibility requirements.  They may be covered by A11Y techniques or accessibility metadata, or even additional specifications.  W3C already has JLreq, ILreq, KLreq, CLreq, and other specs.  There is nothing wrong in having EPUB A11Y J techniques and so forth.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-09-23-epub-a11y#section3) on 2021-09-23

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 3. Informative note for Unicode in EPUB accessibility 

_See github issue [#1790](https://github.com/w3c/epub-specs/issues/1790)._




**Avneesh Singh:** Next topic, unicode  
… I was thinking we should have an informative note in the spec that people should be using UTF-8 because it's required in EPUB and good for accessibility  
… when I speak to people about this, EPUB a11y is the first thing people see  
… come to EPUB core later  
… unicode is important for several languages  
… to solve that query, put in a note  
… Matt has mentioned not knowing where to put it  
… I'm not strong on this idea, but if we can find a spot, it would be nice  

**George Kerscher:** Does EPUBCheck check for UTF-8?  

**Matt Garrish:** Yes  

**George Kerscher:** Isn't this an internationalization issue?  

**Avneesh Singh:** Yes  

> *Bill Kasdorf:* and it's fundamental to XML

**George Kerscher:** I think it would be good to have it mentioned somewhere  

**Avneesh Singh:** Somewhere is the question  

**Matt Garrish:** That's actually what I was going to ask about  
… Makoto has asked for this as well, there's regional specific requirements  
… maybe add a section on internationalization  
… there's some sections on WCAG and EPUB, and there's the techniques  
… we don't clearly say use unicode anywhere  
… it would be an outlier  
… if we want to add a section on i18n, and various standards relating to that, that's one approach  

**John Foliot:** As far as W3C process, horizontal reviews need to be done  
… UTF-8 is a requirement in EPUB?  
… it needs to be captured somewhere  
… if this is something we can discuss in our meeting with Janina  

**Murata Makoto:** I like Matt's idea about introducing a section on i18n  
… if that section is introduced, I will have some things to include  
… TTS of Japanese documents containing ruby  
… DAISY has created a wiki, and Richard Ishida is working on a note  
… i18n WG knows ruby well  
… I pinged him again yesterday about this  
… it's not specific to EPUB, but to HTML  

**Avneesh Singh:** So Matt, if you want a section on i18n, techniques is probably a better place?  

**Matt Garrish:** I don't know if it's a technique  
… it might need to be in the spec  
… specific problems and how to solve them would go in the techniques  
… the core a11y spec has the links to WCAG  
… a brief explainer  

**Avneesh Singh:** That sounds like the way forward  
… a paragraph in the core document  
… and techniques  

**Matt Garrish:** It could even go into the overview  
… I don't think it needs to be elaborate  

**Avneesh Singh:** Action item to draft something and then we can review  

</details>
